utils/build/docker/php/common/install_ddtrace.sh

```diff
@@ -45,10 +45,10 @@ else
 fi
 
 php -d extension=ddtrace.so -d extension=ddappsec.so -r 'echo phpversion("ddtrace");' > \
-  ./SYSTEM_TESTS_LIBRARY_VERSION
+  ./SYSTEM_TESTS_LIBRARY_VERSION 2> /dev/null
 
 php -d extension=ddtrace.so -d extension=ddappsec.so -r 'echo phpversion("ddappsec");' > \
-  ./SYSTEM_TESTS_PHP_APPSEC_VERSION
+  ./SYSTEM_TESTS_PHP_APPSEC_VERSION 2> /dev/null
```

Comment:
why?

Reply:
We have an issue in the AppSec CI, somehow stderr and stdout are being written to the same file, I'm not sure that this will fix it since normally you'd have to explicitly redirect both to stdout for this to happen, but we have no other way of verifying it since it doesn't happen locally.

```
error: Could not open file /opt/datadog/dd-library/appsec-
Warning: PHP Startup: [ddappsec] The following setting datadog.trace.client_ip_header could not be registered due to incompatible versions of ddappsec and ddtrace. This usually happens due to an unsupported or deprecated installation method. Both extensions will continue to work as expected, however the setting will not be taken into consideration. Please have a look at the documentation for more details on the currently supported installation methods: https://docs.datadoghq.com/security_platform/application_security/getting_started/php/?tab=dockercli#get-started in Unknown on line 0
0.4.4/etc/dd-appsec/recommended.json: File name too long

Reply:
I've changed this to use php to supress the error.